backend/src/alembic/versions/fa0b606ae578_add_sourcedocumentdata.py

```diff
@@ -0,0 +1,61 @@
+"""add SourceDocumentData
+
+Revision ID: fa0b606ae578
+Revises:
+Create Date: 2023-11-08 14:42:44.947147
+
+"""
+from typing import Sequence, Union
+
+import sqlalchemy as sa
+from alembic import op
+from sqlalchemy.dialects import postgresql
+
+# revision identifiers, used by Alembic.
+revision: str = "fa0b606ae578"
+down_revision: Union[str, None] = None
+branch_labels: Union[str, Sequence[str], None] = None
+depends_on: Union[str, Sequence[str], None] = None
+
+
+def upgrade() -> None:
+    op.create_table(
+        "version",
+        sa.Column("id", sa.Integer(), nullable=False),
+        sa.Column("version", sa.Integer(), nullable=False),
+        sa.PrimaryKeyConstraint("id"),
+    )
+    op.create_table(
+        "sourcedocumentdata",
```

Comment:
I think it would be cleaner if we import the respective ORM class and use the `__name__` variable (see `backend/src/app/core/data/orm/orm_base.py` instead of hardcoding :-)

Reply:
hardcoding is required. the `__name__` variable might change due to a DB migration (if we ever rename a table)

Reply:
it's generated code, anyway